### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by FlamingB1rd**
Our first PR

### CODE REVIEW

**Comment by ToniAnton22** [comment]
Nice tits

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# burden-of-souls
\ No newline at end of file
+# burden-of-souls
+
+This is a test.
```
Comment (ToniAnton22) [inline README.md:3]: Are you sure?

Comment (ToniAnton22) [reply README.md:3]: ya, that's fine

**Review by ToniAnton22 — APPROVED** [review]
Cool

File: README.md (hunk 1 above)
Comment (FlamingB1rd) [reply README.md:3]: First

**Pull request closed without merging**